Migration to mv3

Jonghakseo opened the pull request:
## Migaration to manifest version 3

https://developer.chrome.com/docs/extensions/develop/migrate/manifest

@N1ck

N1ck commented:
Hey @Jonghakseo thanks for taking a look at this, sorry for being so slow to reply to this. Just wondering if you've tested whether this also works in Firefox? Hopefully I have more time soon I can do a more thorough test!

Jonghakseo commented:
@N1ck 

I added support for firefox.

Added for different support for mv3 itself, which requires a small modification to the manifest after the build.

I've tested it myself, please check it out! And please take a look at the other PRs related to broken UI.

Jonghakseo commented on src/manifest.json:
```diff
@@ -1,50 +1,62 @@
+  "$schema": "https://json.schemastore.org/chrome-manifest",
+  "manifest_version": 3,
+  "name": "GIFs for GitHub",
+  "version": "1.2.11",
```
This probably needs to be bumped up... Please be careful when deploying.

Jonghakseo commented on edit-manifest-for-firefox.js:
https://stackoverflow.com/questions/75043889/manifest-v3-background-scripts-service-worker-on-firefox

N1ck commented:
Thank you! Have included this in https://github.com/N1ck/gifs-for-github/pull/83
